recipes-utils/ultra96-misc/files/wifi.sh

```diff
@@ -18,3 +18,6 @@ ifup wlan0
 # Get IP address
 sleep 1
 udhcpc -i wlan0
+
+# Setup DNS
+echo "nameserver 8.8.8.8" > /etc/resolv.conf
```

Comment:
Yes you're right, it will not work for everybody, but I see this more like an example for customers: 'if you want to change the DNS you can put it here'